components/timeline/src/Timeline.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC, ReactNode, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+
+import '../assets/Timeline.scss';
+
+type TimelineItem = {
+  /* date is used to set the datetime attribute of the <time> element so must
+  be in the ISO format for a date or time. */
```

Comment:
If this is to appear in the docs then it needs to be a JSDoc comment. i.e. `/**` rather than `/*`.

Reply:
Changed to JSDoc in latest commit